index.js

```diff
@@ -28,6 +28,13 @@
 }(this, function () {
   "use strict";
 
+  // Use classList if available, else fall back to manual string comparison
+  var hasClass = document.body.classList
+    ? function(el, className) { return el.classList.contains(className); } 
+    : function(el, className) {
+      return (' ' + el.className + ' ').indexOf(' ' + className + ' ') > -1;
```

Comment:
Support for classList is 89% globally: http://caniuse.com/#search=classList. While high, that's still 1 in 10 people that use browsers that don't support it.